libs/sdk-core/src/lsps0/jsonrpc.rs

```diff
@@ -0,0 +1,99 @@
+extern crate serde;
+extern crate serde_json;
+
+use serde::de::Error;
+use serde::{Deserialize, Deserializer, Serialize};
+use serde_json::Value;
+
+#[derive(Debug)]
+pub enum JsonRpcServerObject {
+    Notification(JsonRpcNotification),
+    Response(JsonRpcResponse),
+    Error(JsonRpcError),
+}
+
+impl<'de> Deserialize<'de> for JsonRpcServerObject {
```

Comment:
Do we need the Deserialize implementation? serde is pretty flexible in deserialization enums (https://serde.rs/enum-representations.html#untagged)

Reply:
Works with untagged. That example in the link is literally jsonrpc ^^

Reply:
Done